mantidproject/mantid
# Remove useless MantidPlot dialog help buttons

**Description of work.**
Changed inheritance from QDialog to MantidDialog. Set useless help buttons to be disabled by default.

**To test:**

1. Open MantidPlot
2. Choose one of the following dialogs to check:
    - Manage User Directories
    - Help -> About MantidPlot
    - Script Window -> Ctrl + F
    - Load Mantid Project
    - View -> Preferences / Manage Custom Menus

Fixes #26134 . 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
